src/util.cpp

```diff
@@ -15,14 +16,22 @@ std::vector<std::string> get_lines(std::filesystem::path const &filename) {
   return ret;
 }
 
-/* std::vector<std::string> separate_substrings(const std::string &str) {
-  std::vector<std::string> ret;
-  std::string word;
-  std::stringstream s_stream(str);
-  while (std::getline(s_stream, word, ' ')) {
-    ret.push_back(word);
+std::unordered_map<std::string, std::string>
+insert_pairs(std::vector<std::string> file_lines) {
+  std::unordered_map<std::string, std::string> key_value_pairs;
+  std::string key;
+  std::string value;
+
+  for (const auto &str : file_lines) {
+    if (auto delimeter_pos = str.find('=');
```

Comment:
confusing line break, see also [this mail](http://lkml.iu.edu/hypermail/linux/kernel/2005.3/08168.html) from Linus as to why you don't have to adhere to the 80-column philosophy - in this case specifically, your code is harder to read because you make these unexpected line breaks